cypress/fixtures/example.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this file necessary?

Reply:
uhm probably not, I think this was automatically added as part of the cypress init script

Reply:
should I delete it?

Reply:
Sure, better not to clutter

Reply:
done